llvm/test/tools/llvm-readobj/ELF/relr-relocs.test

```diff
@@ -240,12 +245,15 @@ Symbols:
 # RUN: llvm-readelf --relocations --raw-relr %t2.has.link 2>&1 | \
 # RUN:   FileCheck -DFILE=%t2.has.link --check-prefix=RAW-GNU2 %s
 
-# RUN: yaml2obj --docnum=3 %s -o %t3
-# RUN: llvm-readelf -r %t3 2>&1 | FileCheck -DFILE=%t3 --check-prefix=GNU3 %s
-
 ## .symtab is invalid. Check we report a warning and print entries without symbolization.
+# RUN: yaml2obj --docnum=3 -DENTSIZE=1 %s -o %t3.err1
+# RUN: llvm-readelf -r %t3.err1 2>&1 | FileCheck -DFILE=%t3.err1 --check-prefixes=GNU3,GNU3-ERR1 %s
```

Comment:
If I'm not mistaken, these aren't actually testing that no symbolization occurred, since there could be other stuff after the checked-for output on each line. You'll need `--match-full-lines`. Same goes for other "not symbolized" cases.

Reply:
Thanks for catching this. Added and added some `-NOT:{{.}}` to enhance tests.